File: e_learning_platform_backend/app/Models/LiveClass.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class LiveClass extends Model
{
    use HasFactory;

    protected $primaryKey = 'LiveClassId';
    protected $fillable = [
        'CourseId', 'Title', 'StartTime', 'EndTime', 'MeetingLink', 'Description', 'Status'
    ];

    // Khóa học chứa lớp học trực tuyến
    public function course()
    {
        return $this->belongsTo(Course::class, 'CourseId', 'CourseId');
    }
}

File: e_learning_platform_frontend/AndroidApp/admin/app/src/main/java/com/seedam/stuonadminrule/domain/model/LiveClass.kt
package com.seedam.stuonadminrule.domain.model

import java.util.Date

data class LiveClass(
    val id: Int,
    val courseId: Int,
    val startTime: Date,
    val endTime: Date,
    val meetingLink: String,
    val description: String?
)
